packages/teleport-uidl-validator/src/uidl-schemas/component.json

```diff
@@ -45,7 +45,7 @@
     "propDefinitions": {
       "type": "object",
       "patternProperties": {
-        ".*": {
+        "^([a-zA-Z0-9])*$": {
```

Comment:
I tried passing special characters too, but our validator is not throwing errors when we pass special characters in UIDL

Reply:
I think patternProperties is acting a bit differently here maybe? what strings have you tried?

Reply:
also, do we need this for the prop definition? since the props we're worried about are usually added from the style rules when using styled-components

Reply:
No, the regex is correct, I tested online and in another schema's it is working fine. we need this to eliminate special-characters being used in the prop name. The issue we encountered using ``background-color`` as a prop name

Reply:
but background-color was not defined in propDefinitions, it is just passed to the styled component to be able to use dynamic values